merge-files.js

```diff
@@ -179,34 +179,52 @@ async function main() {
     config.excludes = userConfig.excludes;
   }
 
-  // Ask choice
-  console.log(`
+ if (projectType === "node") {
+    console.log(`
 Please select one option:
 1) Analyse the source code and provide an overview
 2) Extract the feature list and create a CSV file
 3) Generate a README.MD file
 4) No additional prompt
-`);
+5) Generate import/export relationships
 
-  const choice = await askQuestion("Enter your choice (1-4): ");
-  let tplFile = "";
-
-  switch (choice) {
-    case "1":
-      tplFile = path.join(__dirname, "config/analyze.tpl");
-      break;
-    case "2":
-      tplFile = path.join(__dirname, "config/features.tpl");
-      break;
-    case "3":
-      tplFile = path.join(__dirname, "config/readme.tpl");
-      break;
-    case "4":
-    default:
-      tplFile = path.join(__dirname, "config/prompt.tpl");
-      break;
+`);
+  } else {
+    console.log(`
+Please select one option:
+1) Analyse the source code and provide an overview
+2) Extract the feature list and create a CSV file
+3) Generate a README.MD file
+4) No additional prompt
+`);
   }
 
+const choice = await askQuestion("Enter your choice : ");
+let tplFile = "";
+
+switch (choice) {
+  case "1":
+    tplFile = path.join(__dirname, "config/analyze.tpl");
+    break;
+  case "2":
+    tplFile = path.join(__dirname, "config/features.tpl");
+    break;
+  case "3":
+    tplFile = path.join(__dirname, "config/readme.tpl");
+    break;
+  case "4":
+    tplFile = path.join(__dirname, "config/prompt.tpl");
+    break;
+  case "5":
```

Comment:
@Monikarangasamy - 
a) Why do we have to restrict only for node projects?
b) If we have to restrict, then better not to show the option at all if it is not a node project.